pkg/volume/emptydir/empty_dir_test.go

```diff
@@ -194,6 +195,9 @@ func doTestPlugin(t *testing.T, config pluginTestConfig) {
 	}
 
 	// Stat the directory and check the permission bits
+	if !volumeutil.IsReady(metadataDir) {
```

Comment:
Do you also want to add a test case for calling SetUp again when readydir exists but volume directory does not?